src/constants/npc_constants.asm

```diff
@@ -41,5 +76,120 @@
 	const STEPHANIE_PIC ; $28
 	const AARON_PIC     ; $29
 
-SARA   EQU $1F
-AMANDA EQU $20
+	const_def 1
+	const DRMASON                     ; $01
```

Comment:
I would prefer that these be suffixed with `NPC_` (see #67 )

Reply:
What do you want to do about the ones that aren't exactly npcs? in the upcoming commit, I made things like _11F18 into "NPC_11F18" and also added the prefix to things like "LEGEND_CARD_xxx"

Reply:
I think those should be prefixed with `NPC_` also. They may not be "characters" but they use the NPC system. If these constants don't have prefixes, they are too likely to cause confusion with other constants, especially duelists. I can't think of another prefix that applies equally well to all of them and also provides enough context.
Also, `11F18` is a torch/flame.

Reply:
That's a good point.

Nice! I will rename that as well, thank you.